frontend/src/Styles/Themes/dark.js

```diff
@@ -235,6 +235,7 @@ module.exports = {
   //
   // misc
 
+  progressBarTextColor: white,
```

Comment:
Use hex codes instead of color names.

Reply:
That is calling a previously defined hex code, if you want me to make it all hex I can but kinda sounds like extra work for no change